config/database.yml

```diff
@@ -58,10 +57,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: ruby_community_test
-  username: <%= ENV['DB_USERNAME'] || 'ruby_community' %>
-  password: <%= ENV['DB_PASSWORD'] || 'password' %>
-  host: <%= ENV['DB_HOST'] || '127.0.0.1' %>
+  adapter: sqlite3
```

Comment:
No tests so far and no PostgreSQL specific features used (AFAIK), so IMO being able to run the app with SQLite locally would make things smoother.

But it's definitely debatable.

Reply:
sure, sounds good. I will run it locally tomorrow and most probably we can merge it. thanks!

Reply:
@FabienChaynes sorry to bother you with more and more, but you've started it :D 
Since we're ditching PG from dev and test envs, we should clear a bit more - we don't need `dotenv` gem any more and instructions in Readme became invalid. So

1. 
> Copy the example environment file:
cp .env.development.example .env.development
Edit .env.development and set your database credentials:
DB_USERNAME=your_username
DB_PASSWORD=your_password
Create database, run migrations and seed data:

^^ this should be removed from the Readme file

2. dotenv (gem) may be removed from Gemfile
3. `.env.development.example` file can be removed

but once all that is ready, we'll have a "look how much easier it is with SQLite" PR. 
Could you please add it to the PR so that there are no loose ends after this change?

Reply:
No problem!

Done in b3931ac1bbc67814cc89f95fb8dc583a71896cce and dd6478c176df697f5d670e5847400f916063e2e5.

Reply:
👍